app/swagger/requests/letters.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    class Letters
+      include Swagger::Blocks
+
+      swagger_path '/v0/letters' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Get a list of available letters for a veteran'
+          key :operationId, 'getLetters'
+          key :tags, %w(
+            evss
+          )
+
+          parameter :authorization
+
+          response 200 do
+            key :description, 'Response is OK'
+            schema do
+              key :'$ref', :Letters
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/letters/beneficiary' do
+        operation :get do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Returns service history, and a list of benefit options for use with POST /v0/letters'
+          key :operationId, 'getLettersBeneficiary'
+          key :tags, %w(
+            evss
+          )
+
+          parameter :authorization
+
+          response 200 do
+            key :description, 'Response is OK'
+            schema do
+              key :'$ref', :LetterBeneficiary
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/letters/{id}' do
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Returns a letter as a PDF blob'
+          key :operationId, 'postLetter'
+          key :tags, %w(
+            evss
+          )
+
+          parameter :authorization
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Options to include in generated PDF'
+            key :required, false
+
+            schema do
```

Comment:
`schema` is for inline block definition of a json object within a parameter (in this case the body), `swagger_schema` is for a model.